src/preregister.js

```diff
@@ -4,6 +4,7 @@ import MailchimpInput from "./mailchimp-input.js";
 import CovidPopover from "./covid-popover.js";
 
 var HeroImage = require('./hero.svg');
```

Comment:
We should stick to camelCase for var names

Reply:
Ah, yeah, you're right. I was a careless and just quickly adopted the naming convention for the import statement above